document-store/src/main/java/org/hypertrace/core/documentstore/Query.java

```diff
@@ -19,6 +19,14 @@ public void setFilter(Filter filter) {
     this.filter = filter;
   }
 
+  public void addAllOrderBys(List<OrderBy> orderBys) {
+    this.orderBys.addAll(orderBys);
+  }
+
+  public void addOrderBy(OrderBy orderBy) {
```

Comment:
looks similar to setOrderBy below. Do you want to delete the below one?

Reply:
yeah. let me do that